sccp/figures/figureS1.py

```diff
@@ -21,18 +21,27 @@ def makeFigure():
     # Add subplot labels
     subplotLabel(ax)
 
-    rank = 30
+    rank = 5
 
     (
         lupus_tensor,
-        _,
-        row_colors,
-    ) = load_lupus_data()  # don't need to grab cell types here
+        obs,
+    ) = load_lupus_data(every_n = 10)  # so that it builds quickly
+
 
     weights, factors, _, _ = parafac2_nd(
         lupus_tensor, rank=rank, n_iter_max=20, random_state=1
     )
 
+    status = (
+        obs[["sample_ID", "SLE_status"]]
+        .sort_values(by="sample_ID")
+        .drop_duplicates("sample_ID")
+    )
+
+    lut = {"SLE": "c", "Healthy": "m"}
+    row_colors = status["SLE_status"].map(lut)
+
     plotFactors(
         factors, lupus_tensor, ax[0:3], reorder=(0, 2), trim=(2,), row_colors=row_colors
```

Comment:
@pascoesean if you are gonna label the patients by status, I believe you have to remove the reorder of "0"

Reply:
I _think_ this should be ok-- if the reorder tuple includes 0 then it reorders the row_colors along with the patients :

 if i in reorder:
            X, ind = reorder_table(X)
            yt = yt[ind]
            if i == 0 and not (row_colors is None):
                row_colors = row_colors[ind]

Reply:
^^ common lines 130-134